File: src/Fall20javaProg/TokenType.php
<?php

abstract class TokenType{
    const LBRACKET = 0;
    const RBRACKET = 1;
    const LSQUAREBRACKET = 2;
    const RSQUAREBRACKET = 3;
    const EQUAL = 4;
    const COLON = 5;
    const OUTPUT = 6;
    const SWITCH = 7;
    const CASE = 8;
    const BREAK = 9;
    const DEFAULT = 10;
    const ID = 11;
    const INT = 12;
    const STRING = 13;
    const EOF = 14;
    const OTHER = 15;
}

?>

File: src/Fall20javaProg/TokenType.java
package Fall20javaProg;

public enum TokenType {
      LBRACKET, //{
      RBRACKET, //}
      LSQUAREBRACKET, //[
      RSQUAREBRACKET, //]    
      EQUAL, //=
      COLON, //:
      OUTPUT, //'output'
      SWITCH, //'switch'
      CASE, //'case'
      BREAK, //'break'
      DEFAULT, //'default'
      ID, //[a-zA-Z]+
      INT, //[0-9]+
      STRING, // same as Java string
      EOF, // end of file
      OTHER // anything else
}
